### PR TITLE
Add full screen prop modal

### PR DESCRIPTION
**Pull request by bergholmm**
To more easily be able to achieve [this](https://nordnet.invisionapp.com/d/main?origin=v7#/console/20177685/428042572/preview?scrollOffset=2567.77783203125).

### CODE REVIEW

**Review by victoriatomzik — APPROVED** [review]

File: src/Molecules/Modal/Modal.stories.tsx [hunk 1]
```diff
@@ -406,3 +409,57 @@ export const closeOnBackdropClickStory = () => {
...
+    return (
+      <div>
+        <button type="button" onClick={onOpen}>
+          Open modal
+        </button>
+        <ScrollMaker />
+        <Modal onClose={onClose} title="Dialog information" open={open} fullScreenMobile={false}>
+          <Box mb={2}>
+            <Typography type="primary" as="p">
+              Modals should be used with care as they are quite intrusive on the user experience and
```
Comment (jesperwiner) [inline src/Molecules/Modal/Modal.stories.tsx:434]: Please adjust story to reflect the true intention of "not full screen mobile" with a smaller one like the design with "Are you sure you want to quit" to make it clear what it looks like.

File: src/Molecules/Modal/Modal.stories.tsx [hunk 2]
```diff
@@ -406,3 +409,57 @@ export const closeOnBackdropClickStory = () => {
...
+          </Typography>
+        </Modal>
+      </div>
+    );
+  };
+  return <Example />;
+};
+
+notFullScreenMobile.story = {
+  name: 'Not full width mobile',
```
Comment (jesperwiner) [inline src/Molecules/Modal/Modal.stories.tsx:464]: screen instead of width

**Review by jesperwiner — CHANGES_REQUESTED** [review]

File: src/Molecules/Modal/Modal.stories.tsx (hunk 2 above)
Comment (bergholmm) [reply src/Molecules/Modal/Modal.stories.tsx]: Updated

File: src/Molecules/Modal/Modal.stories.tsx (hunk 1 above)
Comment (bergholmm) [reply src/Molecules/Modal/Modal.stories.tsx]: Updated

**Review by jesperwiner — APPROVED** [review]

**Review by Vikingjonsson — APPROVED** [review]

**Pull request merged**